src/YQPkgPatchList.cc

```diff
@@ -537,7 +537,7 @@ YQPkgPatchCategoryItem::patchCategory( const string & category )
 YQPkgPatchCategory
 YQPkgPatchCategoryItem::patchCategory( const QString & cat )
 {
-    QString category = category.toLower();
```

Comment:
This was a genuine bug that probably I introduced just yesterday: IIRC the parameter was a simple `QString`, and I turned it into a `const QString &`, and then gcc complained about that const thing being changed, so I had to rename the parameter, but I forgot to also change where that parameter was used. My bad.